week-1-ts-ecosystem/w01e04-light-me-up/src/task-1.ts

```diff
@@ -8,40 +8,30 @@
  * Dodatkowo — rzucać błąd, jeśli zapas mocy się wyczerpie.
  * */
 
-class PowerSource {
-  private energySupply = 100
+import { PowerSource } from './power-source';
 
-  consume(energy: number): void {
+class LightBulb {
```

Comment:
`LightBulb` też się powtarza jak `PowerSource` (dawniej). 

Powinno być to w `1` pliku.

Zadania są do siebie podobne - wiec starczy jedna klasa `LightBulb` do `task-1` + `task-2`